apps/spack/build_spack.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
+set -e
 APPS_SPACK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 APP_NAME=spack
 APP_VERSION=0.16.0
 SHARED_APP=${SHARED_APP:-/apps}
-INTEL_MPI_VERSION=${INTEL_MPI_VERSION:-2020.2.254}
+INTEL_MPI_VERSION=${INTEL_MPI_VERSION:-2018.4.274} # This should be retrieved dynamically
```

Comment:
Why change the Intel MPI library to an older version?

Reply:
because 2020 is no longer in the latest images !!!